subscription_events.go

```diff
@@ -0,0 +1,84 @@
+package chartmogul
+
+const subscriptionEventsEndpoint = "subscription_events"
+
+type SubscriptionEvent struct {
+	ID                        uint32      `json:"id,omitempty"`
+	DataSourceUUID            string      `json:"data_source_uuid,omitempty"`
+	CustomerExternalID        string      `json:"customer_external_id"`
+	SubscriptionSetExternalID string      `json:"subscription_set_external_id,omitempty"`
+	SubscriptionExternalID    string      `json:"subscription_external_id,omitempty"`
+	PlanExternalID            string      `json:"plan_external_id,omitempty"`
+	EventDate                 string      `json:"event_date"`
```

Comment:
[blocking] If these (check all fields) don't have `omitempty` and I want to Update the event, what happens is they serialize the null value into the JSON, which I think will result in nullifying it on server or failing the request. You can easily test this, if you write a test for Update.

Reply:
1. The fields without the `omitempty` clause are not nullable on the server, there will be an error message returned.
2. The tests that are connecting to the API do not run on my local machine. AFAIR they never did

Reply:
ad 1: what you say implies if I don't want to update `event_date`, it will always fail.
ad 2: we will need to fix that then.